### ISSUE TITLE
Import Character error

### ISSUE THREAD

**Issue by FunjaPlays** [ClayAmore/ER-Save-Editor#77]
Hey i got this error of i try to Import a Charakter
![image](https://github.com/user-attachments/assets/26d51414-5ea0-4e3e-aaf1-e830440c39e6)

**Comment by ClayAmore** [comment]
Yeah youre trying to load a pre DLC save. They changed the compression library after the DLC to zstd instead of zlib, but you can easily fix it by running the game once, just get to the menu so it updates your regulation file in the save.

**Comment by FunjaPlays** [comment]
Now i have this Problem with my Save file i can opend yesterday !![image](https://github.com/user-attachments/assets/45ebeb1c-2ee5-4b4d-aa34-15dd033821eb)

**Comment by ClayAmore** [comment]
post the save

**Comment by FunjaPlays** [comment]
https://drive.google.com/file/d/1KntRBW23E1DmTMAx2J4NvZ5XA-QzsI2s/view?usp=drivesdk

**Comment by FunjaPlays** [comment]
And i can Not add items to it

**Comment by Gby3** [comment]
Im having issues too when importing a PS4 save to PC... 
When i open the PC file after importing im getting this error:

![ersaveeditor](https://github.com/user-attachments/assets/f7ec4366-1483-4a18-b030-e643481d054a)

the ps4 save:
https://drive.google.com/file/d/13XnR-Ju5SVkv9pvEjKhSocKE7gHjxtC_/view?usp=sharing

**Comment by Buxata** [comment]
> Now i have this Problem with my Save file i can opend yesterday !![image](https://private-user-images.githubusercontent.com/160171321/358187839-45ebeb1c-2ee5-4b4d-aa34-15dd033821eb.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4MDE2NTksIm5iZiI6MTcyMzgwMTM1OSwicGF0aCI6Ii8xNjAxNzEzMjEvMzU4MTg3ODM5LTQ1ZWJlYjFjLTJlZTUtNGI0ZC1hYTM0LTE1ZGQwMzM4MjFlYi5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTZUMDk0MjM5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODBkNmE1ZmJmNjIyYTVmMmU1YjhmYmQxNTEyM2UzNWJkYmI5YmY3MWFjZGYwOWZmYTIxN2M4MmQxMGY2YjQ2YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Uv3QIbtjEqnQhvVVweey9dJFKpO5PBUj_TQtwnH12F0)

I've got the same issue, need a save?
I'm running it on linux so I am manually building the latest release.

**Comment by FunjaPlays** [comment]
I Need only my ps4 Save to be edited 
But i can Not add items to it 
If i Click on a item to add nothing Happens

**Comment by FunjaPlays** [comment]
Some News @ClayAmore

**Comment by FunjaPlays** [comment]
@Buxata hey can u edit my save file ?
